kt/godot-library/src/main/kotlin/godot/gen/godot/Area.kt

```diff
@@ -66,7 +66,7 @@ open class Area : CollisionObject() {
    * `local_shape` the index of the [godot.Shape] of this Area used by the [godot.PhysicsServer].
    */
   val areaShapeEntered: Signal4<Long, Area, Long, Long> by signal("area_id", "area", "area_shape",
-      "local_shape")
+      "self_shape")
```

Comment:
also in the current class ref these params are named `local_shape`
https://docs.godotengine.org/en/3.3/classes/class_area.html#signals
So i think the generation is not correct yet.

Reply:
it seems so, le me re dot it